tests/lib/shared/deep-merge-arrays.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-undefined -- `null` and `undefined` are different in options */
```

Comment:
I believe the intent of `no-undefined` is to let you write `void 0` instead of `undefined`. It won't restrict you from using `null`.

Reply:
`void 0` is ... not a practice I'm used to anybody preferring 😄. I'm unclear on what the suggested next step is.

Reply:
We use `void 0` instead of `undefined` in the rest of the repo, so we could do the same thing in this file.

Reply:
it's typically/historically preferred to ack that `undefined` can be redefined, and `void 0` is shorter than `void undefined` (altho i've taken to using the latter since it's more readable and probably equally minifiable)